tests/unit/Broadway/ThirdPartyPayloadAndMetadataDomainMessageSerializerTest.php

```diff
@@ -84,15 +84,76 @@ public function it_should_deserialize_to_a_DomainMessage()
             ->andReturn($payload);
 
         $serialized = [
-            'id'         => 'a',
-            'playhead'   => 0,
-            'metadata'   => ['metadata'],
-            'payload'    => ['payload'],
+            'id' => 'a',
+            'playhead' => 0,
+            'metadata' => ['metadata'],
+            'payload' => ['payload'],
             'recordedOn' => (new \DateTime('2015-01-01'))->format(DateTime::FORMAT_STRING)
         ];
 
         $deserialized = $serializer->deserialize($serialized);
 
         $this->assertEquals($event, $deserialized);
     }
+
+    /**
+     * @test
+     * @expectedException \Burrow\Serializer\DeserializeException
+     * @dataProvider getMalformedDomainMessageExample
+     */
+    public function it_should_failed_to_deserialize_to_a_DomainMessage($serialized)
```

Comment:
failed -> fail

it fails when trying to deserialize a DomainMessage from a malformed message ?